.github/workflows/scan-distribution.yaml

```diff
@@ -24,3 +24,43 @@ jobs:
       registry: ghcr.io/controlplaneio-fluxcd/${{ matrix.variant }}
     secrets:
       pull-token: ${{ secrets.GHCR_READONLY }}
+
+  scan-operator:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+    - name: Latest release
+      id: release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        VERSION=$(gh release view --repo controlplaneio-fluxcd/flux-operator --json tagName -q .tagName)
+        echo version=${VERSION} >> $GITHUB_OUTPUT
+    - name: Export image
+      id: image
+      run: echo fo=${REGISTRY}:${VERSION} >> $GITHUB_OUTPUT
+      env:
+        REGISTRY: ghcr.io/controlplaneio-fluxcd/flux-operator
```

Comment:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable